packages/beroot/PKGBUILD

```diff
@@ -0,0 +1,32 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=beroot
+pkgver=1.0.1.r101.gc1ce228
+pkgrel=1
+pkgdesc='A post exploitation tool to check common misconfigurations to find a way to escalate our privilege.'
+arch=('any')
+groups=('blackarch' 'blackarch-exploitation')
+url='https://github.com/AlessandroZ/BeRoot'
+license=('custom:unkown')
```

Comment:
https://github.com/AlessandroZ/BeRoot/issues/36

We can't add copyrighted tools

Reply:
so....should i remove its PKGBUILD ?

Reply:
The best would have been that you made 3 PR for the 3 tools and we would have let this one as WIP until a license is set upstream. You can maybe remove this one from this PR and add the PKGBUILD in a comment of the issue so someone would be able to PR when upstream would be ready.